test/mechanics/mesh/UnitMeshFem.cpp

```diff
@@ -28,6 +28,36 @@ void Check2DMesh(NuTo::MeshFem& mesh)
     BOOST_CHECK_NO_THROW(transformedMesh.NodeAtCoordinate(Eigen::Vector2d(43., 9.)));
 }
 
+BOOST_AUTO_TEST_CASE(MeshTrusses)
+{
+    auto mesh = NuTo::UnitMeshFem::CreateTrusses(13);
+    BOOST_CHECK_EQUAL(mesh.Elements.Size(), 13);
+    BOOST_CHECK_EQUAL(mesh.Nodes.Size(), 14);
+
+    bool nodesHaveExpectedValue = true;
+    for (unsigned int i = 0; i < mesh.Nodes.Size(); ++i)
+    {
+        if (std::abs(mesh.Nodes[i].GetValues()[0] - static_cast<double>(i) / mesh.Elements.Size()) > 1e-6)
```

Comment:
You rely on `operator[]` and that the nodes are somehow in the same order as you created them. I would not require that... 
.